code_examples/sdk_examples/src/core_features/public_credentials/02_create_credential.ts

```diff
@@ -0,0 +1,23 @@
+import * as Kilt from '@kiltprotocol/sdk-js'
+
+export function createNftCollectionCredential(
+  ctype: Kilt.ICType,
+  assetDid: Kilt.AssetDidUri,
+  artistDid: Kilt.DidUri
+): Kilt.IPublicCredentialInput {
+  const claims: Kilt.IClaimContents = {
+    name: 'Awesome NFT drop',
+    // NFT collection only has 100 pieces in total
+    pieces: 100,
+    // NFT collection was released on Jan 1st, 2023
+    creationDate: new Date(2023, 0, 1).toISOString(),
+    artistIdentity: artistDid
+  }
+
+  return {
+    claims,
+    cTypeHash: Kilt.CType.idToHash(ctype.$id),
+    delegationId: null,
```

Comment:
Delegations are only one form of authorization. Maybe it's the only form right now but shouldn't this be called authorization to be generic enough for new methods?

Reply:
That comes from the SDK. Right now it's called `delegationId`. But you are right. We will change that to `authorizationId` once we have more features or anyway before we go v1.